kart/diff_util.py

```diff
@@ -166,6 +167,7 @@ def get_dataset_diff(
 def get_file_diff(
```

Comment:
FYI: In workdir.py, in the reset() function, it's possible - at the place where the Datasets objects are obtained - to obtain RepoStructure objects there as well. If we did that, we wouldn't have to change this function here.

However, I can't really provide a strong reason why that way is obviously better than this way... so I just added this comment while I think about it a bit more